Backpack/src/main/java/net/skyscanner/backpack/bottomnav/BpkBottomNav.kt

```diff
@@ -46,7 +47,7 @@ open class BpkBottomNav @JvmOverloads constructor(
     super.setOnNavigationItemSelectedListener(it)
   }
 
-  private val fontSpan = BottomNavSpan(context)
+  private val fontSpan = BottomNavSpan(context.withBpkThemeInEditMode(isInEditMode))
```

Comment:
we need to apply the theme attribute when in edit mode for some components for the preview to work